Cargo.toml

```diff
@@ -34,10 +34,12 @@ categories = ["command-line-utilities", "development-tools"]
 
 [dependencies]
 aho-corasick = "1.0.2"
+async-dup = "1.2.4"
 backoff = { version = "0.4.0", default-features = false }
 camino = "1.1.4"
 clap = { version = "4.3.2", features = ["derive", "wrap_help", "env", "string"] }
 command-group = { version = "2.1.0", features = ["tokio", "with-tokio"] }
+dyn-clone = "1.0.16"
```

Comment:
I don't know how `dyn-clone` works, but it's `dtolnay` so I trust it lol.

Reply:
Removed in https://github.com/MercuryTechnologies/ghciwatch/pull/175/commits/508bcf98521987a80928ab2f0570b2fcbdc71a81, it's no longer needed after https://github.com/MercuryTechnologies/ghciwatch/pull/175/commits/145f5ec71e0ebebd850f544c1816c1dd528ef87b